dashboard/client/api/kube-api.ts

```diff
@@ -40,25 +41,67 @@ export interface IKubeApiLinkBase extends IKubeApiLinkRef {
 }
 
 export class KubeApi<T extends KubeObject = any> {
-  static parseApi(apiPath = ""): IKubeApiLinkBase {
+  static parseApi(apiPath = "") {
     apiPath = new URL(apiPath, location.origin).pathname;
+
     const [, prefix, ...parts] = apiPath.split("/");
     const apiPrefix = `/${prefix}`;
 
     const [left, right, found] = split(parts, "namespaces");
```

Comment:
Not part of this PR changes, but can we rename `found` as `namespaced`. Would make code a bit more readable. WDYT?

Reply:
That makes sense. There is the one case under `namespaced === true` where we figure out that it isn't but I think that the comment should help with any misunderstanding.